MOBILE_API.xml

```diff
@@ -732,7 +754,11 @@
         <element name="mediaTrack">
             <description>The track field of NGN and GEN1.1 MFD displays. This field is only available for media applications; applies to "Show"</description>
         </element>
-        
+
+        <element name="templateTitle" since="6.0">
+            <description>The title of the new template that will be displayed; applies to "Show"</description>
+        </element>
+
```

Comment:
This PR should only include widget related changes. Please remove this element from the enum.

Reply:
fixed in 086a52f7d5dcd3505f62478018294a3bd5165a4f